services/QuillLMS/engines/evidence/spec/controllers/evidence/feedback_controller_spec.rb

```diff
@@ -208,53 +210,53 @@ module Evidence
       it 'should return successfully' do
         post("plagiarism", :params => ({ :entry => "No plagiarism here.", :prompt_id => prompt.id, :session_id => 1, :previous_feedback => ([]) }), :as => :json)
         parsed_response = JSON.parse(response.body)
-        expect(true).to(eq(parsed_response["optimal"]))
+        expect(parsed_response["optimal"]).to(eq(true))
       end
 
       it 'should return 404 if prompt id does not exist' do
         post("plagiarism", :params => ({ :entry => "No plagiarism here.", :prompt_id => 100, :session_id => 1, :previous_feedback => ([]) }), :as => :json)
-        expect(404).to(eq(response.status))
+        expect(response.status).to(eq(404))
       end
 
       it 'should return successfully when there is plagiarism that matches the first plagiarism text string' do
         post("plagiarism", :params => ({ :entry => ("bla bla bla #{plagiarized_text1}"), :prompt_id => prompt.id, :session_id => 1, :previous_feedback => ([]) }), :as => :json)
         parsed_response = JSON.parse(response.body)
-        expect(false).to(eq(parsed_response["optimal"]))
+        expect(parsed_response["optimal"]).to(eq(false))
```

Comment:
Is it preferred to use `be false` vs `eq false`? (I don't have a strong opinion here)

Reply:
I think it depends on how restrictive you want the matching to be.  I tend to favor strict matching so I'll change this to be false.

Reply:
Some of the cops, like `RSpec/ExpectActual` are enabled by [default](https://docs.rubocop.org/rubocop-rspec/cops_rspec.html#rspecexpectactual) so we don't have to explicitly reference them in the config.